plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -471,6 +483,48 @@ public MongoCursor<Document> execute(MongoTableHandle tableHandle, List<MongoCol
         return iterable.iterator();
     }
 
+    private static Set<String> findDbRefColumns(MongoCollection<Document> collection)
```

Comment:
Is it possible to maintain if a column is of `DBRef` type in MongoColumnHandle so that we don’t have guess the datatype once again. 

Cc: @ebyhr